test/simulator/test/org/apache/cassandra/simulator/test/HarrySimulatorTest.java

```diff
@@ -504,43 +527,38 @@ public void shutdown()
     /**
      * Simulation entrypoint; syntax sugar for creating a simulation.
      */
-    static void simulate(Consumer<ClusterSimulation.Builder<HarrySimulation>> configure,
-                         Consumer<IInstanceConfig> instanceConfigUpdater,
-                         Configuration.ConfigurationBuilder harryConfig,
-                         String[] properties,
-                         Function<HarrySimulation, ActionSchedule.Work[]>... phases) throws IOException
+    void simulate(Consumer<ClusterSimulation.Builder<HarrySimulation>> configure,
+                  Consumer<IInstanceConfig> instanceConfigUpdater,
+                  Configuration.ConfigurationBuilder harryConfig,
+                  String[] properties,
+                  Function<HarrySimulation, ActionSchedule.Work[]>... phases) throws IOException
     {
         try (WithProperties p = new WithProperties().with(properties))
         {
             HarrySimulationBuilder factory = new HarrySimulationBuilder(harryConfig, instanceConfigUpdater);
 
             SimulationRunner.beforeAll();
-            long seed = System.currentTimeMillis();
-            // Development seed:
-            //long seed = 1687184561194L;
-            System.out.println("Simulation seed: " + seed + "L");
+            long seed = SimulationRunner.parseHex(Optional.ofNullable(this.seed)).orElseGet(() -> new Random().nextLong());
+            logger.info("Seed 0x{}", Long.toHexString(seed));
```

Comment:
this is for consistency... having this test do seeds differently than the other tests makes it harder for people to jump in and see what's going on.